packages/frontend/services/store/slices/anotherExampleTempState.ts

```diff
@@ -0,0 +1,16 @@
+import { TTempState as TAnotherTempState } from "./tempExampleState";
+
+export type TAnotherTempState2 = {
+  tempStuff: string;
+};
```

Comment:
I guess we don't need that for this example?

Seems we are importing the Type from the other example

Reply:
i fixed it, was a mistake